src/main/java/org/asciidoctor/ExportRenderer.java

```diff
@@ -1,3 +1,18 @@
+/*
+ * Copyright 2013-2015 John Ericksen
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
Good, in this case we can just configure the plugin to use /* instead.
Added the block below to the plugin configuration:
<mapping>
   <java>SLASHSTAR_STYLE</java>
</mapping>